src/locales/ar.json

```diff
@@ -96,6 +96,11 @@
     "title": "ألعاب"
   },
   "games": {
+    "quizgame": {
+      "title": "لعبة الاسئلة",
+      "screenshot": "https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7"
```

Comment:
this doesn't have to be in the json. unless its different per translation

just declare a variable `const QUIZ_GAME_IMAGE = <the url>` and use it in code

Reply:
also you added these translations but where are they used?

Reply:
actually i added this image and title so tarkan can use them in the game page, we will have a game page has the 3 games with this screenshot of each game.

Reply:
removed the screenshot